### PR TITLE
fix: hide action button for bot service profile (AR-2011)

### PR DESCRIPTION
**Pull request by Garzas**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-2011" title="AR-2011" target="_blank">BugAR-2011</a>  I should not be able to send a connection request to a bot or service
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

Because currently opening conversation with bot is not implemented, action buttons in bot user profile will be hidden

### Dependencies (Optional)

_If there are some other pull requests related to this one (e.g. new releases of frameworks), specify them here._

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-816/1/) **failed**.

File: app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileScreen.kt [hunk 1]
```diff
@@ -184,14 +186,22 @@ fun OtherProfileScreenContent(
...
-                            onCancelConnectionRequest,
-                            acceptConnectionRequest,
-                            ignoreConnectionRequest
-                        )
+                        // TODO show open conversation button for bots after AR-2135
+                        if (state.membership == Membership.Service) {
+//                            WirePrimaryButton(
+//                                text = stringResource(R.string.label_open_conversation),
+//                                onClick = onOpenConversation,
+//                            )
```
Comment (yamilmedina) [inline app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileScreen.kt:194]: I would be better if we just remove this commented code :D

File: app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileState.kt [hunk 2]
```diff
@@ -16,7 +17,8 @@ data class OtherUserProfileState(
     val phone: String = "",
     val connectionStatus: ConnectionStatus = ConnectionStatus.Unknown,
     val membership : Membership = Membership.None,
-    val groupState: OtherUserProfileGroupState? = null
+    val groupState: OtherUserProfileGroupState? = null,
+    val botService: BotService? = null,
```
[the remark this answers is not in the document]
Comment (Garzas) [reply app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileState.kt:21]: In future this model will contain also provider name which is needed in this view

File: app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileScreen.kt (hunk 1 above)
Comment (Garzas) [reply app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileScreen.kt:194]: Sure

**Review by saleniuk — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-816/2/) **failed**.

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-816/3/) **failed**.

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-816/4/) **failed**.

**Comment by AndroidBob** [comment]
Build [5](https://10.10.124.134/job/wire-android-reloaded/job/PR-816/5/) **failed**.

**Comment by AndroidBob** [comment]
Build [6](https://10.10.124.134/job/wire-android-reloaded/job/PR-816/6/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4063722)(4063722)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4063722)(4063722)-dev-release.apk)

**Pull request merged**